lib/routes/api/admin/index.js

```diff
@@ -10,8 +10,16 @@ var router = express.Router();
 
 router.use(auth.requireAdminOr401);
 
-/*
- * GET /admin/users
+/**
+ * @api {get} /admin/users Get All Users
+ * @apiPermission SiteAdmin
+ * @apiDescription Retrieves a list of all Strider users.
+ * @apiName AdminGetUsers
```

Comment:
Not sure if `Admin[functionality]` is redundant since it's part of the `Admin` group. Thoughts?